templates/crowdstrike-block.hbs

```diff
@@ -1,72 +1,189 @@
     {{#if detection.behaviors}}
-        <div class="p-title">{{fa-icon "brain" fixedWidth=true}} Behavior Information
-            <span class="p-action" {{action "showAllBehaviorInfo" detectionIndex}}>
-                {{#if detection.__showAllBehaviorInfo}}
-                    &nbsp;(view compact {{fa-icon "caret-down"}})
-                {{else}}
-                    &nbsp;(view all {{fa-icon "caret-up"}})
-                {{/if}}
-            </span>
-        </div>
-        {{#if detection.__showAllBehaviorInfo}}
-            {{#each detection.behaviors as | behavior behaviorIndex|}}
-                <div class="p-title">Behavior ID: {{behavior.behavior_id}}</div>
-                {{object-to-table behavior}}
-            {{/each}}
-        {{else}}
-            {{#each detection.behaviors as | behavior behaviorIndex|}}
-                <div class="p-title">Behavior: {{behavior.behavior_id}}</div>
-                {{#each compactBehaviorProperties as |prop|}}
-                    {{#if (get behavior prop)}}
-                        <div>
-                            <span class="p-key">{{humanize prop}}:</span>
-                            <span class="p-value">{{get behavior prop}}</span>
-                        </div>
-                    {{/if}}
-                {{/each}}
-            {{/each}}
-        {{/if}}
+      <div class='p-title'>{{fa-icon 'brain' fixedWidth=true}}
+        Behavior Information
+        <span class='p-action' {{action 'showAllBehaviorInfo' detectionIndex}}>
+          {{#if detection.__showAllBehaviorInfo}}
+            &nbsp;(view compact
+            {{fa-icon 'caret-down'}})
+          {{else}}
+            &nbsp;(view all
+            {{fa-icon 'caret-up'}})
+          {{/if}}
+        </span>
+      </div>
+      {{#if detection.__showAllBehaviorInfo}}
+        {{#each detection.behaviors as |behavior behaviorIndex|}}
+          <div class='p-title'>Behavior ID: {{behavior.behavior_id}}</div>
+          {{object-to-table behavior}}
+        {{/each}}
+      {{else}}
+        {{#each detection.behaviors as |behavior behaviorIndex|}}
+          <div class='p-title'>Behavior: {{behavior.behavior_id}}</div>
+          {{#each compactBehaviorProperties as |prop|}}
+            {{#if (get behavior prop)}}
+              <div>
+                <span class='p-key'>{{humanize prop}}:</span>
+                <span class='p-value'>{{get behavior prop}}</span>
+              </div>
+            {{/if}}
+          {{/each}}
+        {{/each}}
+      {{/if}}
     {{/if}}
-{{/each}}
+  {{/each}}
+{{/if}} 
 
+{{#if (eq activeTab 'crowdstrikeIoc')}}
+  {{#if details.devices}}
+    <div>
+      Displaying
+      {{details.devices.length}}
+      of
+      {{details.meta.totalResults}}
+      results.
+    </div>
+    <div class='p-title'>{{fa-icon 'laptop' fixedWidth=true}}
+      Device Information
+    </div>
+    {{#each details.devices as |device|}}
+      <div class='devices-container'>

```

Comment:
We should include a link to the device similar to how we do it above with detections.

```
    <div class='p-title'>
      {{fa-icon 'laptop' fixedWidth=true}}
      Device Information
      <a href='{{device.__url}}' class='p-link detection-title'>View Device in Falcon
        {{fa-icon 'external-link-square' fixedWidth=true class='external-link'}}</a>
    </div>
```